doc-theme/theme.scss

```diff
@@ -66,15 +76,15 @@ $list-group-color: $primary !default;
 }
```

Comment:
ugh I was hoping I could comment on code I hadn't changed...

Anyhow, the transition property seems unnecessary? as far as I can tell, the navbar doesn't _do_ anything. And its not obvious to me (because I don't generally work in Quarto) that the `navbar-*` variables are functional, but in testing it appears they are so 👍

Reply:
I think that you are correct- most likely a copy/paste issue since the nav on the main site does transition when you scroll, nothing fancy like that here.